lightning/src/ln/channelmanager.rs

```diff
@@ -1863,6 +1897,28 @@ impl<Signer: Sign, M: Deref, T: Deref, K: Deref, F: Deref, L: Deref> ChannelMana
 		}
 	}
 
+	/// Send a spontaneous payment, which is a payment that does not require the recipient to have
+	/// generated an invoice. Optionally, you may specify the preimage. If you do choose to specify
+	/// the preimage, it must be a cryptographically secure random value that no intermediate node
+	/// would be able to guess -- otherwise, an intermediate node may claim the payment and it will
+	/// never reach the recipient.
+	///
+	/// Similar to regular payments, you MUST NOT reuse a `payment_preimage` value. See
+	/// [`send_payment`] for more information about the risks of duplicate preimage usage.
```

Comment:
This also needs to mention that you should see `send_payment` for more details on the return value. Further, this needs to mention that the `Route` must only have at most one path. Finally, we need to actually check that, but not here - as far as I can see we don't currently require that non-payment-secret-containing sends don't contain more than one path, which we should do in `send_payment_internal`.